artem-serbin/README.md

```diff
@@ -0,0 +1,45 @@
+Google Maps
+===========
+
+
+
+Google Maps is a widely used web mapping service developed by Google. It provides users with detailed information about geographical locations and allows them to explore various places around the world. This README file provides an overview of Google Maps and its features.
+
+## Features
+* Interactive Maps: Google Maps offers interactive maps that allow users to zoom in and out, pan, and explore different areas. Users can view maps in different modes such as satellite view, terrain view, and street view.
+
+* Search Functionality: Users can search for specific addresses, landmarks, businesses, and points of interest on Google Maps. The search functionality provides suggestions and auto-complete options to make it easier for users to find what they are looking for.
+
+* Directions: Google Maps provides detailed directions for driving, walking, cycling, and public transit. Users can enter their starting point and destination to get step-by-step instructions, estimated travel times, and alternate routes.
+
+* Real-Time Traffic Updates: Google Maps offers real-time traffic information, including congestion, accidents, and road closures. This feature helps users plan their routes and avoid heavy traffic.
+
+* Street View: Street View allows users to explore and navigate through 360-degree street-level imagery. Users can virtually walk through streets, view landmarks, and even enter certain businesses.
+
+* Public Transit Information: Google Maps provides information on public transportation systems in many cities around the world. Users can see bus, train, and subway schedules, routes, and transit fares.
+
+* Satellite Imagery: Google Maps offers high-resolution satellite imagery that allows users to see aerial views of locations worldwide. This feature is particularly useful for exploring remote areas and checking out landscapes.
+
+* Local Business Listings: Users can find local businesses, read reviews, view photos, and get contact information through Google Maps. It provides a platform for businesses to showcase their offerings and connect with potential customers.
+
+## Getting Started
+To start using Google Maps, follow these steps:
+
+1. Open a web browser and go to [Google Maps](https://www.google.com/maps).
+2. Use the search bar to enter a location, address, or point of interest.
+3. Navigate the map by zooming in and out, panning, or switching to different map modes.
+4. Click on markers, businesses, or points of interest to view more information.
+5. Use the directions feature to get driving, walking, or transit directions between locations.
+6. Explore Street View by dragging the yellow Pegman icon onto the map and dropping it onto a street.
```

Comment:
Just use Pegman instead of 'the yellow Pegman icon' (see https://support.google.com/maps/answer/3093484?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Cuse-google-search%2Cuse-pegman).